### PR TITLE
add druid client

### PR DESCRIPTION
**Pull request by danfrankj**
@matthewwardrop Druid now supports SQL

### CODE REVIEW

File: omniduct/databases/druid.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        self.__druid = None
+
+    # Connection
+    def _connect(self):
+        from pydruid.db import connect
+        logger.info('Connecting to Druid database ...')
+        self.__druid = connect(self.host, self.port, path='/druid/v2/sql/', scheme='http')
+
+    def _is_connected(self):
+        return hasattr(self, '__druid') and self.__druid is not None
```
Comment (matthewwardrop) [inline omniduct/databases/druid.py]: `hasattr` doesn't seem to be necessary here, given that it is explicitly set in the __init__ method.

File: omniduct/databases/druid.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    def _disconnect(self):
+        logger.info('Disconnecting from Druid database ...')
+        try:
+            self.__druid.close()
+        except Exception:
```
Comment (matthewwardrop) [inline omniduct/databases/druid.py]: Do you know what the Druid client raises if it is not connected?

File: omniduct/databases/druid.py (hunk 1 above)
Comment (matthewwardrop) [inline omniduct/databases/druid.py]: Does `druid` have a way to check if it is connected?

Comment (danfrankj) [reply omniduct/databases/druid.py]: udpated!

File: omniduct/databases/druid.py (hunk 2 above)
Comment (danfrankj) [reply omniduct/databases/druid.py]: It's sometimes buggy so I'm going to leave this as is for now
https://github.com/druid-io/pydruid/pull/119

**Pull request merged**